src/server/index.jsx

```diff
@@ -166,12 +166,13 @@ server
         } catch ({ message }) {
           logger.error(`Error fetching Cosmos dials: ${message}`);
         }
+        // Preserve initial dial state in window so it is available during hydration
+        data.dials = dials;
```

Comment:
Could the dials object not be added to the window in the same we add the initial page data [here](https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L96)

Reply:
It could, what benefit do you see to that? I was trying to keep everything within `SIMORGH_DATA` to avoid polluting the window any more than is strictly needed.

Reply:
Only for consistency really. Happy either way.

Reply:
Apologies I completely misread what was happening here. This is fine.